ajax/libs/surface/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "surface",
+  "version": "1.0.1",
+  "description": "Material Design, CSS only framework.",
+  "main": "gulpfile.js",
```

Comment:
@bennypowers We don't need `"main"`, `"scripts"`, `"bugs"`, `"devDependencies"` field. Let's remove it.